app/models/auth/expirable.rb

```diff
@@ -0,0 +1,7 @@
+module Auth
+  module Expirable
+    def expiration
+      (Time.now.to_i + Rails.configuration.keystorm['expiration_window']).to_s
```

Comment:
I would prefer:
```ruby
def expiration
  Time.now + expiration_window # no need to convert the result to string or time to Integer
end

def expiration_window
  Rails.configuration.keystorm['expiration_window'].to_i # catch configuration mistakes early
end
```

Reply:
Interesting. When I run tests with this change I get that all expiration are set to `2001-09-09 09:46:40.000000000 +0000` instead of `1000028800`. Should I convert to_i in `OIDC` or `VOMS` classes?

Reply:
Good point :-) You can convert the result to Integer, if the rest of the code expects that.